grpc.sh

```diff
@@ -14,44 +16,26 @@ incremental_recipe: |
 ---
 #!/bin/bash -e
 
-if [[ $PKGVERSION =~ ^v((.+)(-alice[0-9]+)|(.+))$ ]]; then
-  # Define a variable with the version number minus the `-alice` suffix.
-  # Works also with non-ALICE versions
-  REALVERSION=${BASH_REMATCH[2]}
-  [[ $REALVERSION ]] || REALVERSION=${BASH_REMATCH[4]}
-else
-  echo "Invalid version number: $PKGVERSION"
-  exit 1
-fi
-
-# We need the sources here; do not copy everything by using `--reference` wisely
-git clone "$SOURCE0" --reference "$SOURCEDIR" .
+pushd $SOURCEDIR
 git checkout "$GIT_TAG"
 git submodule update --init
+popd
 
-# gRPC has a custom Makefile which readily breaks with some environment vars,
-# better to run it in a clean environment
-env -i HOME="$HOME"                             \
-       LC_CTYPE="${LC_ALL:-${LC_CTYPE:-$LANG}}" \
-       PATH="$PATH"                             \
-       USER="$USER"                             \
-       LD_LIBRARY_PATH="$LD_LIBRARY_PATH"       \
-       make ${JOBS:+-j$JOBS} prefix=$INSTALLROOT
-
-# ldconfig won't work if we're not running make install as root, and that's ok,
-# we don't need it
-sed -i.deleteme -e 's/ldconfig/true/' Makefile
-rm -f *.deleteme
-
-make prefix=$INSTALLROOT install
+opts=(
```

Comment:
What's the point of this? Can't you simply put it inline?

Reply:
No point except having a list of things where you don't have to remember to add a `\` at the end of each line.

Reply:
Do you want me to change it to inline?

Reply:
Yes, please, let’s stay consistent and keep it simple.